gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageStatistics.java

```diff
@@ -95,7 +96,42 @@ public enum GoogleCloudStorageStatistics {
   GCS_API_SERVER_TIMEOUT_COUNT(
       "gcs_api_server_timeout_count",
       "Counts the occurrence of server side 504 error status code",
-      TYPE_COUNTER);
+      TYPE_COUNTER),
+
+  GCS_BACKOFF_TIME("gcs_backoff_time", "Total back off time in milliseconds", TYPE_COUNTER),
+
+  GCS_BACKOFF_COUNT(
+      "gcs_backoff_count",
+      "Counts the number of times a request was retried after backing off due to some retriable response",
+      TYPE_COUNTER),
+
+  GCS_EXCEPTION_COUNT(
+      "gcs_exception_count", "Counts the occurence of exception from GCS APIs", TYPE_COUNTER),
+
+  GCS_API_TIME(
+      "gcs_api_time", "Tracks the amount of time spend while calling GCS APIs.", TYPE_COUNTER),
+
+  GCS_METADATA_REQUEST("gcs_metadata_request", "Tracks GCS GET metadata API calls.", TYPE_DURATION),
+
+  GCS_GET_MEDIA_REQUEST("gcs_get_media_request", "Tracks GCS GET data API calls", TYPE_DURATION),
+
+  GCS_LIST_DIR_REQUEST(
+      "gcs_list_dir_request", "Tracks GCS GET list prefix API calls", TYPE_DURATION),
+
+  GCS_LIST_FILE_REQUEST(
+      "gcs_list_file_request",
+      "Tracks GCS GET list file API calls. This is usually called with '/' as delimiter with some prefix and maxResults as 1",
+      TYPE_DURATION),
+  GCS_CONNECTOR_TIME(
```

Comment:
this seems to be capturing hcfs level api time , should this even be tracked in `gcsio`?

Reply:
Make sense. Updated.